packages/app/app/containers/LibraryViewContainer/index.js

```diff
@@ -31,10 +32,12 @@ function mapStateToProps(state) {
   }
 
   return {
+    tracks: state.local.direction === 'ascending' ? tracks : tracks.reverse(),
+    filterApplied: tracks.length < unfilteredTracks.length,
```

Comment:
why not just use `!_.isEmpty(state.local.filter`?

Reply:
That works too. Depends on if you want an empty library with someone typing a filter into it to receive the same message "Library is empty" or rather "No search results" (despite there being no library contents to begin with). I figured the first behavior is a bit more useful (emphasizing the empty library), but either works (since both messages are accurate).

Want me to change it?

Reply:
No, after consideration your way of handling it is better.